src/main/webapp/jsp/SignupPage.jsp

```diff
@@ -0,0 +1,124 @@
+              <div class="card-body text-black text-center">
+
+                <form id="signupForm" action="./signup" method="POST" onsubmit="return validateForm()">
+                  <div class="d-flex align-items-center mb-3 pb-1">
+                    <i class="fas fa-cubes fa-2x me-3" style="color: #ff6219;"></i>
+                    <span class="h2 fw-bold mb-0">TOMTickets</span>
+                  </div>
+
+                  <h5 class="fw-normal mb-3" style="letter-spacing: 1px;">Registra un nuovo profilo</h5>
+                  <div class="row">
+                    <div class="col-md-6 mb-2">
+                      <div data-mdb-input-init class="form-outline">
+                        <input type="text" id="nome" name="nome" class="form-control" />
+                        <label class="form-label" for="nome">Nome</label>
+                      </div>
+                    </div>
+                    <div class="col-md-6 mb-2">
+                      <div data-mdb-input-init class="form-outline">
+                        <input type="text" id="cognome" name="cognome" class="form-control" />
+                        <label class="form-label" for="cognome">Cognome</label>
+                      </div>
+                    </div>
+                  </div>
+
+                  <div class="row">
+                    <div class="col-md-6 mb-2">
+                      <div data-mdb-input-init class="form-outline">
+                        <input type="date" id="dataNascita" name="dataNascita" class="form-control" />
+                        <label class="form-label" for="dataNascita">Data di nascita</label>
+                      </div>
+                    </div>
+                    <div class="col-md-6 mb-2">
+                      <div data-mdb-input-init class="form-outline">
+                        <input type="tel" id="telefono" name="telefono" class="form-control" />
+                        <label class="form-label" for="telefono">Numero di telefono</label>
+                      </div>
+                    </div>
+                  </div>
+
+                  <div data-mdb-input-init class="form-outline mb-2">
+                    <input type="email" id="email" name="email" class="form-control" />
+                    <label class="form-label" for="email">Indirizzo Email</label>
+                  </div>
+
+                  <div class="row">
+                    <div data-mdb-input-init class="form-outline mb-2">
+                      <input type="text" id="username" name="username" class="form-control" />
+                      <label class="form-label" for="username">Username</label>
+                    </div>
+                  </div>
+
+                  <div class="row">
+                    <div class="col-md-6 mb-2">
+                      <div data-mdb-input-init class="form-outline">
+                        <input type="getPasswordState()" id="password" name="password" class="form-control" />
```

Comment:
set this as required

Reply:
What the fine is "getPasswordState()"? use "password"
Indeed, getPasswordState() does not work well with the "makePasswordvisible()" script, it has inverted values